### PR TITLE
Fix 'not enough arguments' during wasmtime run

### PR DESCRIPTION
**Pull request by yurydelendik**
Fixes tiny regression after #789 

When `cargo run` is used for wasm file without interface types, the following message is observed:

```
Error: failed to run main module `...`

Caused by:
    not enough arguments for 
```

### CODE REVIEW

**Comment by alexcrichton** [comment]
Whoops!

Could a test be added for this?

**Comment by yurydelendik** [comment]
Added simple CLI to cover the issue.

File: tests/cli_tests.rs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#[test]
+fn run_wasmtime_simple() -> Result<()> {
+    run_wasmtime(&["run", "tests/wasm/simple.wasm", "--invoke", "simple", "4"])
+}
+
+// Wasmtime shakk when not enough arguments were provided.
+#[test]
+fn run_wasmtime_simple_fail_no_args() {
+    assert!(
+        run_wasmtime(&["run", "tests/wasm/simple.wasm", "--invoke", "simple"]).is_err(),
```
Comment (alexcrichton) [inline tests/cli_tests.rs]: Could the `*.wasm` file be avoided getting checked into the repository? If necessary we could use the `wat` crate to build the `*.wasm` dynamically and write that out to a file to run, but ideally I think we wouldn't check in any `*.wasm` files here.

**Comment by kubkon** [comment]
Just a note that this PR fixes #858 :-)

**Comment by alexcrichton** [comment]
Hm instead of using a build script, could this just use the `wat` crate on the fly during testing?

**Comment by alexcrichton** [comment]
👍

**Pull request merged**